src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -766,6 +766,9 @@ table,
 .item-left {
 	padding-right: 1.25rem;
 }
+.item-left > * {
+	vertical-align: middle;
```

Comment:
Should this use "middle" or "baseline"? It seems like the latter would work better given that the fonts may be somewhat different.

Reply:
Middle:

![Screenshot from 2022-12-10 18-22-10](https://user-images.githubusercontent.com/3050060/206867379-9fed1dd4-1de8-4909-9d02-f6eceefcc956.png)

Baseline:

![image](https://user-images.githubusercontent.com/3050060/206867371-7895e148-5120-4355-8fdb-5ed158391c04.png)

I tend to prefer the "middle" alignment because "baseline" renders weirdly since even though they share the same baseline, they don't have the same height, making it a bit weird for the eye (imo). For "middle", they are vertically aligned which I find more pleasant.

Reply:
Well in short, I have a preference for the "middle" alignment but don't have a strong opinion about it so as is preferred.

@notriddle: I updated the selector to make it more specific.